Vagrantfile

```diff
@@ -35,7 +35,8 @@ def setup_host
 end
 
 # Make sure the host computer is set up every time a vagrant command is run
-setup_host
+# Comment this back in if you need Kerberos authentication
+# setup_host
```

Comment:
I'm not sure that this should be commented out by default. Having Kerberos authentication is useful when testing features involving LDAP and the only way to login without Kerberos would be to use the master password.

I like what you did with the NFS flag, maybe something similar could be done with Kerberos? You could make a `use_vpn` flag that defaults to true and can be toggled by individual developers.